### DISCUSSION TITLE
[Git Cleanup] Propose Useless Branches for Removal

### DISCUSSION THREAD

**Discussion by Mirrorgo** [DemocracyDevelopers/AuditVisualizer#39] [Git Management]
If you think your branch is no longer needed, you can list the branch name here. Mirrorgo (Aaron) will regularly delete these remote branches.

**Comment by lvyl9909** [comment]
Can u delete the branch named feature/tutorial-page? 3q.

**Reply by Mirrorgo** [reply]
Done!

**Comment by lvyl9909** [comment]
And test/yilin_test2. 3q

**Reply by Mirrorgo** [reply]
done

**Comment by Auroraphile** [comment]
feature/test-kwj
feature/dashboard-kwj

**Reply by Mirrorgo** [reply]
done
